### PR TITLE
C# index & variables are updated

### PR DESCRIPTION
**Pull request by SijoGeorge7012**
image
image
image
image
image
image
image
image

### CODE REVIEW

File: 002variables.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+			<div class="slides">
+				<section>
+                    <h2>C# VARIABLES</h2>
+					<center>
+                    <p>  * Variables are containers for storing data values.
+						<br>  * A variable is a name given to a storage area that is used to store values of various data types.
+					</center>
+                     
+                    </p>
+                </section>
```
Comment (wandercode) [inline 002variables.html:26]: include a table as a new slide of different types of variables (include the previous table created)

**Review by wandercode — CHANGES_REQUESTED** [review]
each variable should have a to do task

**Review by rohithkrajan — APPROVED** [review]

**Review by wandercode — APPROVED** [review]

File: 001syntsx.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
+<!doctype html>
```
Comment (wandercode) [inline 001syntsx.html:1]: change file name to correct spelling

**Review by wandercode — CHANGES_REQUESTED** [review]
try to arrange the code well aligned

File: 001syntsx.html (hunk 2 above)
Comment (SijoGeorge7012) [reply 001syntsx.html:1]: changed file name
please review

**Comment by wandercode** [comment]
LGTM

**Review by wandercode — APPROVED** [review]

**Pull request merged**